proposed/http-client/http-client-meta.md

```diff
@@ -36,6 +36,19 @@ specify the default behaviours.
 
 ## Approaches
 
+### Default behavior
+
+HTTP clients may be configured by the application author, ie to set a default timeout or to 
```

Comment:
> HTTP clients may be configured by the application author

This is a little bit confusing for me together with the default timeout thingy. Also this section does not really explain what default behaviour means.

So I would start with the fact, that default behaviour mean that we don't recommend/restrict (?) HTTP Client developers to define default behaviour for the following [...] (Although I saw that list somewhere)

Then I would say why we do this (Liskov) and then mention that it's the user responsibility to full configure a client. If someone wants to provide some default configuration stuff, use a client factory.

So this is kind of a reverse of the current content logic.

The alternative looks good to me.

WDYT?